lib/features/recode/config.dart

```diff
@@ -187,6 +196,10 @@ class RecodeConfigState extends ConsumerState<RecodeConfig> {
       isNumeric = ref.read(typesProvider)[selected] == Type.numeric;
     }
 
+    // When cleanse is disabled, chr variables can be recodec categoric.
```

Comment:
CLEANSE is **only** applicable on a datasaet load. I don;t think it should be tested here? Why so?